test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-sale-product-set @ git+https://github.com/trobz/sale-workflow@14.0-mig-sale_product_set#subdirectory=setup/sale_product_set
```

Comment:
Hello @phuctranerp 

Can you remove this line since the dependency has been merged ? 

Thanks !

Reply:
done thanks